src/main/terrier.cpp

```diff
@@ -26,7 +28,11 @@ int main() {
     terrier::storage::init_storage_logger();
     terrier::transaction::init_transaction_logger();
     terrier::parser::init_parser_logger();
+    terrier::network::init_network_logger();
 
+    terrier::network::TerrierServer terrier_server;
+
+    terrier_server.SetupServer().ServerLoop();
```

Comment:
These two lines (setting up the TerrierServer), should be after this try...catch block.
The try...catch block should be only for logger initialization.